pkgs/development/python-modules/tpm2-pytss/default.nix

```diff
@@ -20,50 +19,17 @@
 
 buildPythonPackage rec {
   pname = "tpm2-pytss";
-  version = "2.1.0";
+  version = "2.2.1";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
```

Comment:
FYI we can also just fetch from GitHub. There is often really no point fetching something from pypi